include/uv-nuttx.h

```diff
@@ -98,7 +98,9 @@ ssize_t writev(int __fd, const struct iovec* __iovec, int __count);
 int getpeername(int sockfd, struct sockaddr* addr, socklen_t* addrlen);
 
 // Maximum queue length specifiable by listen
+#if !defined(SOMAXCONN)
```

Comment:
I'd prefer `#ifndef SOMAXCONN` in favor of style consistency. Good to me otherwise

Reply:
You are right, thanks ;)